tasks/benchmark/Cargo.toml

```diff
@@ -0,0 +1,16 @@
+[package]
+description = "rspack benchmark tooling"
+edition     = "2021"
+license     = "MIT"
+name        = "rspack_benchmark"
+repository  = "https://github.com/web-infra-dev/rspack"
+version     = "0.1.0"
+publish = false
+
+[dependencies]
+criterion = { package = "codspeed-criterion-compat", version = "2.7.2" }
```

Comment:
I think we need to run both remotely and locally. Would you please add a criterion package here as well?

Reply:
done by being replaced by `criterion2`